packages/palette/src/elements/Marquee/Marquee.tsx

```diff
@@ -55,19 +62,22 @@ const Item = styled.div`
 export interface MarqueeProps extends BoxProps {
   variant?: keyof typeof VARIANTS
   speed?: string
+  mobileSpeed?: string
   color?: string
   marqueeText: string
+  textSize?: TextVariant
   divider?: boolean
 }
 
 export const Marquee: React.FC<MarqueeProps> = ({
   marqueeText,
+  textSize = "sm",
   divider = true,
-  speed = "10s",
+  speed = "30s",
+  mobileSpeed = "10s",
```

Comment:
A better API would be to match the responsive props array style. So `speed={["10s", "30s"]}

Reply:
It might be a bit tricky to implement but basically you'd loop over it in the `Inner` component. I can help with that if it proves ellusive.

Reply:
👍